tests/unit/CIGitLogicTest.sh

```diff
@@ -389,10 +390,10 @@ update_staging_from_main
 tag_staging
 
 # Verify production release list
-assert_prs_merged_between '1.0.0-2' '1.0.1-4' "[ 9, 7, 6, 5, 2 ]"
+assert_prs_merged_between '1.0.0-2' '1.0.1-4' "[ 9, 7, 6, 5, 3, 2 ]"
 
 # Verify PR list for the new checklist
-assert_prs_merged_between '1.0.1-4' '1.0.2-0' "[ 10, 8 ]"
+assert_prs_merged_between '1.0.1-4' '1.0.2-0' "[ 10, 9, 8 ]"
```

Comment:
This is because the PR 9 has been CPed [here](https://github.com/Expensify/App/blob/5b487e465742fc8b9f244f0204bb286612f41141/tests/unit/CIGitLogicTest.sh#L358)